OIPA/iati/models.py

```diff
@@ -1734,7 +1734,6 @@ class CrsAddLoanTerms(models.Model):
         null=True,
         blank=True,
         default=None, on_delete=models.CASCADE)
-    repayment_plan_text = models.TextField(null=True, blank=True, default="")
```

Comment:
Is there a specific reason why you removed this field ? Also, I can see that currently this is not used anywhere (this is probably the reason), but in such database-level changes you should still add a migration file in the commit

Reply:
Yes, according to specifications, free tex is no longer allowed in this element. That’s why I removed the text field in the database. Again, you’re right. I should have added the migration file. I will do in on Monday. 😁